### ISSUE TITLE
Songtext translation

### ISSUE THREAD

**Issue by xstable** [zaps166/QMPlay2#742]
I found that songtext-tab shows a translation down behind the first (english) translation. 
My system language is German, but the translation is some Cyrillic (maybe Russian language?)

![image](https://github.com/zaps166/QMPlay2/assets/1778068/6f866ed1-1fca-4328-99a2-95905b948df0)

**Comment by zaps166** [comment]
QMPlay2 doesn't translate anything, the translation comes from the website. To translate lyrics I have to use DeepL or Google Translate or other web service.

Where do you see Cyrillic? :astonished:

**Comment by xstable** [comment]
Is there any setting to edit this translation-page which is used in the background?
Where exactly does it come from?

**Comment by zaps166** [comment]
No. In the past I had two lyrics websites, but one is already down.

**Comment by zaps166** [comment]
The website I use: https://www.tekstowo.pl/

**Comment by xstable** [comment]
Is there a way to use other Webpages (like https://www.songtexte.com/) , or is it internally hard coded?

**Comment by zaps166** [comment]
It's currently hard-coded, maybe I'll do it in the future.
